### DISCUSSION TITLE
Hi, is neovide configurable with Rust?

### DISCUSSION THREAD

**Discussion by Joe23232** [neovide/neovide#526] [Q&A] [answered]
Hi, is neovide configurable with Rust as well and Rust is used to "script" the configuration of neovide?

**Comment by Kethku** [comment, marked as answer]
I'm not sure what you mean. Could you give some more details? Are you trying to use the neovide front end for a project unrelated to neovim? There has been some discussion about how to do a similar thing in the future, but at the moment the two are linked pretty closely.

@j4qfrost did some work to use the project in a separate context, but since then the project has become more tightly coupled. I would like to eventually extract the monospace renderer parts of neovide out into a separate library which neovide just depends on, but thats a pretty big change which I haven't spent the time to fully hammer out yet.

**Reply by Joe23232** [reply]
> I'm not sure what you mean. Could you give some more details? Are you trying to use the neovide front end for a project unrelated to neovim? There has been some discussion about how to do a similar thing in the future, but at the moment the two are linked pretty closely.


I mean like something like DWM where every configuration is done through the source code. Is this a thing with neovide by any chance?

**Reply by Kethku** [reply]
Again, I don't really understand. Neovide is configured from your neovim config. The configuration isn't really done in the rust source code, but rather in the lua or vimscript initialization scripts.

**Comment by Joe23232** [comment]
I see, thanks mate
